### PR TITLE
Make commands_with_underscores work

### PR DESCRIPTION
**Pull request by franciscouzo**

### CODE REVIEW

File: b2/console_tool.py [hunk 1]
```diff
@@ -1447,6 +1447,8 @@ def get_parser(cls):
     def run_command(self, argv):
         signal.signal(signal.SIGINT, keyboard_interrupt_handler)
         parser = self.get_parser()
+        if len(argv) >= 2:
+            argv[1] = argv[1].replace('_', '-')
```
Comment (mlech-reef) [inline b2/console_tool.py:1451]: In python3 we could use aliases. Maybe add a comment to use it if we drop support for python2

Comment (mlech-reef) [reply b2/console_tool.py:1451]: Another way is to monkypatch argparse for python2 and add a functionality for aliases which is pretty straightforward

Comment (ppolewicz) [reply b2/console_tool.py:1451]: I think this makes `b2 __help` work. This monkeypatching seems like a better idea tbh. @franciscouzo will you try it? Talk to @mlech-reef if you need guidance

Comment (mlech-reef) [reply b2/console_tool.py:1451]: @franciscouzo Currently I'm working on other improvements and fixes like more descriptive `--help`, better handling argument errors, etc. I had to subclass `argparse.ArgumentParser` anyway in `arg_parser.py:ArgumentParser`. You can do the same, so we will have less merge conflicts.

Comment (mlech-reef) [reply b2/console_tool.py:1451]: An also please sync your code with `Backblaze:master`. CI has been fixed and now AppVeyor and Travis should be green before the merge.

Comment (mlech-reef) [reply b2/console_tool.py:1451]: Please see #629

Comment (ppolewicz) [reply b2/console_tool.py:1451]: @mlech-reef was command with underscores a part of #629 or does it still need some work? Will you take care of it?

Comment (mlech-reef) [reply b2/console_tool.py:1451]: @ppolewicz #635
Please close this one

**Comment by ppolewicz** [comment]
This is continued in https://github.com/Backblaze/B2_Command_Line_Tool/pull/635

**Pull request closed without merging**